cnxarchive/database.py

```diff
@@ -52,7 +52,8 @@ def _read_sql_file(name):
     'get-module-versions': _read_sql_file('get-module-versions'),
     'get-subject-list': _read_sql_file('get-subject-list'),
     'get-featured-links': _read_sql_file('get-featured-links'),
-    }
+    'get-users-by-ids':_read_sql_file('get-users-by-ids'),
+    } 
```

Comment:
I'm just going to paste pep8 errors.  Please fix them.  We hope to be pep8 compliant eventually... :)

```
cnxarchive/database.py:55:23: E231 missing whitespace after ':'
cnxarchive/database.py:56:6: W291 trailing whitespace
```